pallets/messages/src/mock.rs

```diff
@@ -54,10 +55,37 @@ impl system::Config for Test {
 }
 
 parameter_types! {
-	pub const MaxMessagesPerBlock: u16 = 500;
+	pub const MaxMessagesPerBlock: u32 = 500;
 	pub const MaxMessageSizeInBytes: u32 = 100;
 }
 
+impl std::fmt::Debug for MaxMessagesPerBlock {
```

Comment:
These traits was needed in tests. So I had to implement them

Reply:
I dont know why we need this. When we actually need to use it we do T:: MaxMessagesPerBlock::get().
so I needed to print something out
```
let a  = T:: MaxMessagesPerBlock::get()
println!("hello {:?}", a);
```

Reply:
It does not work in tests because it needs the `Debug` trait to be implemented in case you are checking 2 objects inside a `assert_eq!` And I'm checking these objects quality inside `add_message_should_store_message_on_temp_storage`

Reply:
Actually we just need those only for `MaxMessageSizeInBytes` so I'm removing the `MaxMessagesPerBlock` implementations